examples/Atlas/+atlasControllers/AtlasPlanEvalAndControlSystem.m

```diff
@@ -70,7 +70,15 @@
         if ~obj.quiet
           t0 = tic();
         end
-        qp_input = obj.plan_eval.getQPControllerInput(t, x);
+        qp_input_obj = obj.plan_eval.getQPControllerInput(t, x);
+        if isa(qp_input_obj, 'atlasControllers.QPInputConstantHeight')
```

Comment:
consider re-ordering this logic?  isnumeric is probably much faster than isa for some class.